contracts/IdentityOracle.sol

```diff
@@ -202,37 +160,27 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
         }
         return result;
     }
-
-    function setJobIDIPFS1(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID1 = _jobid;
-    }
-
-    function setJobIDIPFS2(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID2 = _jobid;
-    }
-
-    function setJobIDStateHash(string memory _jobid) public {
+    
+    function getJobIDStateHashIPFSCID() public view returns (string memory) {
```

Comment:
it doesnt make sense to make a view function  onlyAvatar
in any case,  simply make the variables public so there's no need for a view function that simply returns a variable

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/f05ce99726a869f1dfaf944456647138eaf25fa3